autoshop/core/admin.py

```diff
@@ -1,6 +1,9 @@
-# -*- coding: utf-8 -*-
-from __future__ import unicode_literals
-
 from django.contrib import admin
+from core.models import Profile
+
+
+class ProfileAdmin(admin.ModelAdmin):
+    list_display = ['user', 'bio', 'gender', 'date_of_birth']
```

Comment:
remove bio` for list display as some bio's could be too large to display in list view.